interview-questions.md

```diff
@@ -8,50 +8,51 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is the difference between a parameter and an argument?
 
-Your answer:
+Your answer: A parameter is a place holder and an argument is the actual data.
 
 Researched answer:
 
 2. The JavaScript built in method .map() takes predefined parameters. What are they? Which are required and which are optional?
 
-Your answer:
+Your answer: Value (required), Index (optional), Array (optional), order matters! Value is required because wthout value there can be no function.
 
-Researched answer:
+Researched answer: Value (required), Index (optional), Array (optional), order matters! Value is required because wthout value there can be no function.
 
 3. What is the difference between map and filter?
 
-Your answer:
+Your answer: Map iterates through each element and returns a new array of the same length, trading one piece of data for another. Filter does what the name suggests and filters through data returning a subset or just a part of the original array. 
 
-Researched answer:
+Researched answer:  Map iterates through each element and returns a new array of the same length, trading one piece of data for another. Filter does what the name suggests and filters through data. Filter needs an evaluation and it can be anything that returns a Boolean value. This process allows Filter to return a subset or just a part of the original array.
 
-1. What is iteration?
+4. What is iteration?
 
-Your answer:
+Your answer: The process performing a function a set certain number of times until a specific action is met. 
 
-Researched answer:
+Researched answer: The process performing a function a set certain number of times until a specific action is met. 
 
-1. What is the difference between a function and a method?
+5. What is the difference between a function and a method?
 
-Your answer:
+Your answer: A function is reusuable and  can hold the method. 
+Method is a part of the functionality of the actual function. 
```

Comment:
I believe you're confusing a method with a built-in method. A function is a reusable block of code, but a method is a function that is associated with an object. If you recall the objects lectures, we can create a method within an object. That is what is known as a method in JavaScript.